src/index.html

```diff
@@ -5,4 +5,151 @@
+            <h2>Modern</h2>
+        </li>
+        <li class="tap">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Ftap.png?v=1605143407247" alt="Tap">
+            <h2>Tap</h2>
+        </li>
+        <li class="jazz">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fjazz.png?v=1605143407596" alt="Jazz">
+            <h2>Jazz</h2>
+        </li>
+        <li class="hiphop">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fhiphop.png?v=1605143407749" alt="Hip-Hop">
+            <h2>Hip-Hop</h2>
+        </li>
+        <li class="african">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fafrican.png?v=1605143399790" alt="African">
+            <h2>African</h2>
+        </li>
+        <li class="little-ones">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Flittleones.png?v=1605143402975" alt="little-ones">
+            <h2>Little Ones</h2>
+        </li>
+        <li class="repertory">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Frepertory.png?v=1605143401041" alt="repertory">
+            <h2>Repertory</h2>
+        </li>
+        <li class="adult-classes">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fadultclasses.png?v=1605143403559" alt="dult Classes">
+            <h2>Adult Classes</h2>
+        </li>
+    </ul>
+</div>
+<div class="Register">
+    <h1>2020 Fall Registration</h1>
+    <button class="accordion">Step 1</button>
+    <div class="panel">
+        <p>
+            CURRENT STUDENTS: Check your email! You will have class placements sent to you. If you want to add a class you have not taken before, please contact us before registering. <br>
+            PROSPECTIVE STUDENTS: Read our class descriptions, then call or email us to discuss appropriate class placement.
+        </p>
+    </div>
+    <button class="accordion">Step 2</button>
+    <div class="panel">
+        <p>
+            REVIEW CALENDAR, TUITION, REGISTRATION PROCEDURES<br>
+            Double ​check the classes you would like to take. Some classes meet twice per week. Make note of any potential conflicts and talk to us if you have one.
+            Commit to the year or join us for our 13-Week Session! Save money by paying in full. You will otherwise be put on a monthly payment plan.</p>
+    </div>
+
+    <button class="accordion">Step 3</button>
+    <div class="panel">
+        <p>
+            REGISTER ONLINE<br>
+            Log in to your account, or create a new one!
+            Once you've selected your classes, we will review and approve your registration.​ We automatically charge a deposit for registration fee, September tuition, and costume deposit(s) to secure your space in the classes you've selected. You will then be sent a confirmation email and balance due for the season.</p>
```

Comment:
Same note as above for this content. I would expect to see something like this instead:
```suggestion
        <h3>Register Online</h3>
        <p>Log in to your account, or create a new one!</p>
        <p>Once you've selected your classes, we will review and approve your registration.​ We automatically charge a deposit for registration fee, September tuition, and costume deposit(s) to secure your space in the classes you've selected. You will then be sent a confirmation email and balance due for the season.</p>
```

(The heading level may be different depending on the structure you end up with here; this assumes the `h1` for this section becomes an `h2`).